From 37613b67ba52b5ad4e81d7ea38adc04027d9f9e5 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 23 May 2019 09:25:51 +0200
Subject: [PATCH] alsa: handle alsa-lib 1.1.9

alsa-lib 1.1.9 removed /usr/include/alsa from the include path, we
must include <alsa/asoundlib.h>
---
 spa/plugins/alsa/alsa-monitor.c | 2 +-
 spa/plugins/alsa/alsa-sink.c    | 2 +-
 spa/plugins/alsa/alsa-source.c  | 2 +-
 spa/plugins/alsa/alsa-utils.h   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/spa/plugins/alsa/alsa-monitor.c b/spa/plugins/alsa/alsa-monitor.c
index d8935cd7..16a01302 100644
--- a/spa/plugins/alsa/alsa-monitor.c
+++ b/spa/plugins/alsa/alsa-monitor.c
@@ -25,7 +25,7 @@
 #include <poll.h>
 
 #include <libudev.h>
-#include <asoundlib.h>
+#include <alsa/asoundlib.h>
 
 #include <spa/support/log.h>
 #include <spa/support/type-map.h>
diff --git a/spa/plugins/alsa/alsa-sink.c b/spa/plugins/alsa/alsa-sink.c
index c31fe3e2..3c252fab 100644
--- a/spa/plugins/alsa/alsa-sink.c
+++ b/spa/plugins/alsa/alsa-sink.c
@@ -19,7 +19,7 @@
 
 #include <stddef.h>
 
-#include <asoundlib.h>
+#include <alsa/asoundlib.h>
 
 #include <spa/node/node.h>
 #include <spa/param/audio/format.h>
diff --git a/spa/plugins/alsa/alsa-source.c b/spa/plugins/alsa/alsa-source.c
index 74bbb3c0..8efc8fde 100644
--- a/spa/plugins/alsa/alsa-source.c
+++ b/spa/plugins/alsa/alsa-source.c
@@ -19,7 +19,7 @@
 
 #include <stddef.h>
 
-#include <asoundlib.h>
+#include <alsa/asoundlib.h>
 
 #include <spa/node/node.h>
 #include <spa/utils/list.h>
diff --git a/spa/plugins/alsa/alsa-utils.h b/spa/plugins/alsa/alsa-utils.h
index 5ba57bc5..3b590007 100644
--- a/spa/plugins/alsa/alsa-utils.h
+++ b/spa/plugins/alsa/alsa-utils.h
@@ -26,7 +26,7 @@ extern "C" {
 
 #include <stddef.h>
 
-#include <asoundlib.h>
+#include <alsa/asoundlib.h>
 
 #include <spa/support/type-map.h>
 #include <spa/support/loop.h>
